src/elements/asset.rs

```diff
@@ -526,13 +526,14 @@ fn chain_asset_stats_delta<T>(
     start_height: usize,
     apply_fn: AssetStatApplyFn<T>,
 ) -> (T, Option<BlockHash>) {
+    let headers = chain.store().headers();
     let history_iter = chain
         .history_iter_scan(b'I', &asset_id.into_inner()[..], start_height)
         .map(TxHistoryRow::from_row)
         .filter_map(|history| {
-            chain
-                .tx_confirming_block(&history.get_txid())
-                .map(|blockid| (history, blockid))
+            // skip over entries that point to non-existing heights (may happen during reorg handling)
+            let header = headers.header_by_height(history.key.confirmed_height as usize)?;
```

Comment:
Liquid can have, at most, one reorged block. Not sure if it's worth changing this part.

If we do want to handle it, couldn't the `confirmed_height` be stale? Another block could have re-orged the tip. 

This does save a lookup in `tx_confirming_block` (`self.store.history_db.get(&TxConfRow::key(txid))?;`) though since we can get the confirmed_height directly